.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+# Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
"List any dependencies that are required for this change."

I'm not really sure what this means. Is this `cargo` dependencies? Thanks!

Reply:
Honestly this means nothing. I'll remove it.